data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -114,13 +116,14 @@ podman run -d --name coreos-downloader \
 podman wait -i 1000ms coreos-downloader
 
 podman run -d --net host --privileged --name httpd \
-     --restart on-failure \
-     --env IRONIC_RAMDISK_SSH_KEY="$IRONIC_RAMDISK_SSH_KEY" \
-     --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
-     -v $IRONIC_SHARED_VOLUME:/shared:z --entrypoint /bin/runhttpd ${IRONIC_IMAGE}
+    --restart on-failure \
+    --env IRONIC_RAMDISK_SSH_KEY="$IRONIC_RAMDISK_SSH_KEY" \
+    --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
+    --env HTTP_PORT=$DEFAULT_HTTP_PORT \
+    -v $IRONIC_SHARED_VOLUME:/shared:z --entrypoint /bin/runhttpd ${IRONIC_IMAGE}
 
 # Add firewall rules to ensure the IPA ramdisk can reach httpd, Ironic and the Inspector API on the host
-for port in 80 5050 6385 ; do
+for port in 6180 5050 6385 ; do
```

Comment:
I'm no longer convinced this is the right thing to do. If it has always been on port 80, as seems to be the case, then it needs to stay there as there are likely customers with external firewalls who haven't opened port 6180 to the bootstrap, and moving the port would break them.

Reply:
we could leave port 80 open temporarily but we should correct this anyway
we can maybe fix the docs for 4.9 and 4.10, but then leave the change for 4.11 ?
the docs say that port 6180 should be the only one needed